embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/session/orchestrator/SessionOrchestratorImpl.kt

```diff
@@ -110,6 +125,20 @@ internal class SessionOrchestratorImpl(
             val timestamp = clock.now()
             sessionService.endSessionWithCrash(timestamp, crashId)
             backgroundActivityService?.endBackgroundActivityWithCrash(timestamp, crashId)
+            logger.logDebug("Session ended with crash")
         }
     }
+
+    private fun logSessionStateChange(
+        sessionId: String?,
+        timestamp: Long,
+        inBackground: Boolean,
+        stateChange: String
```

Comment:
I assume this is just something you need for logging and not a concept you want to leverage more later?

Reply:
Just logging for now. Ideally I'd like to keep a record of all session state changes in some sort of buffer for debugging purposes, but that will come later.